store/LocalStore.js

```diff
@@ -4,21 +4,20 @@ const { LruMap: Lru } = require('toad-cache')
 
 function LocalStore (cache = 5000, timeWindow, continueExceeding) {
   this.lru = new Lru(cache)
-  this.timeWindow = timeWindow
   this.continueExceeding = continueExceeding
 }
 
-LocalStore.prototype.incr = function (ip, cb, max, ban) {
+LocalStore.prototype.incr = function (ip, cb, timeWindow, max, ban) {
```

Comment:
This will be a breaking change since you are modifying the signature of the increment method, this method is called with custom store implementations too, so they would need to be modified

Another change that modifies this method's signature is queued up in the `next` branch ([it removed the `ban` parameter from `incr`](https://github.com/fastify/fastify-rate-limit/commit/fe1ca6f2fd7d16f590fb34ca8a48acdc508ad293)), I suggest releasing these changes together in the next major that will support Fastify v5